src/components/Competition/RolesManager/RolesManager.js

```diff
@@ -88,7 +93,7 @@ const RolesManager = ({ wcif, onWcifUpdate }) => {
                 <TextField
                   label="Search"
                   value={searchString}
-                  onChange={event => setSearchString(event.target.value)}
+                  onChange={event => handleSearchStringChange(event.target.value)}
```

Comment:
Nit: for the sake of convention, I'd suggest passing the whole event to the handler and just do `onChange={handleSearchStringChange}` =)

Reply:
Thanks for the advice. React is really new to me, so I'm not familiar with best practices yet.